data/campaigns/Under_the_Burning_Suns/scenarios/11_Battle_for_Zocthanol_Isle.cfg

```diff
@@ -518,17 +609,52 @@
             side=1
         [/filter]
 
-        [message]
-            speaker=Zhul
-            message= _ "Ugh. That jungle seems dark and foreboding. Do you see those eyes? There are things staring at us from the depths of the jungle, but I can’t tell what they are. As much as I love plants and trees, I don’t like the look of it."
-        [/message]
+        [if]
+            [have_unit]
+                ability = natures_aid
+                [or]
+                    ability = extra_natures_aid
+                [/or]
+            [/have_unit]
+            [then]
+                [message]
+                    speaker=Zhul
+                    message= _ "The jungle here does't want to answer me, as if afraid of indescribable horrors within it. As much as I love nature, this one here wants to make me scared."
```

Comment:
Spelling: doesn’t (including apostrophe)

I'd consider nature as uncountable, so suggest: 'As much as I love nature, this place frightens me.'